### ISSUE TITLE
`pyi-makespec` ignores `--log-level` argument

### ISSUE THREAD

**Issue by bersbersbers** [pyinstaller/pyinstaller#8174]
### Context information (for bug reports)

* Output of `pyinstaller --version`: ```6.3.0```
* Version of Python: 3.12.1
* Platform: Windows
* How you installed Python: `scoop`
* Did you also try this on another platform? No

* try the latest development version? same thing

* follow *all* the instructions in our "If Things Go Wrong" Guide
  * [x] start with clean installation
  * [x] use the latest development version
  * [n/a] Run your frozen program **from a command window (shell)** — instead of double-clicking on it
  * [n/a] Package your program in **--onedir mode**
  * [n/a] Package **without UPX**, say: use the option `--noupx` or set `upx=False` in your .spec-file
  * [n/a] Repackage you application in **verbose/debug mode**. For this, pass the option `--debug` to `pyi-makespec` or `pyinstaller` or use `EXE(..., debug=1, ...)` in your .spec file.

### A minimal example program which shows the error

```
touch empty.py
pyi-makespec empty.py --log-level ERROR
```

### Stacktrace / full error message
```
Wrote C:\Code\project\empty.spec.
Now run pyinstaller.py to build the executable.
```

There should not be any output since no errors occured.

**Comment by bwoodsend** [comment]
That's not logging though – it's just a print statement. 

Since it doesn't do any logging, I don't think it makes sense for `pyi-makespec` to accept a `--log-level` option. I'd rather block that option+command combination and have people `pyi-makespec ... > /dev/null` (that command never prints anything else) than start guessing generic verbosity levels from logging levels.

**Comment by rokm** [comment]
> That's not logging though – it's just a print statement.
> 
> Since it doesn't do any logging, I don't think it makes sense for `pyi-makespec` to accept a `--log-level` option. I'd rather block that option+command combination and have people `pyi-makespec ... > /dev/null` (that command never prints anything else) than start guessing generic verbosity levels from logging levels.

I was going to say the same - none of the programs from `PyInstaller.utils.cliutils` really make use of `logging` (and the stuff they print via `print()` is more along the lines of command output than messages that belong to categorized logging). I also think we should remove `--log-level` option from `pyi-makespec`.

**Comment by XiaoZhi-Python-China** [comment]
Well, W feels that there is nothing wrong with what the official said. You can understand it this way. pyi-makespec is used to generate configurations, such as. spec files, while --log-level is used to generate logs, which will be printed through some output results. When you use it, it is only a hard-coded statement printed by print. When you use PyInstaller-one dir xx. spec, you will use this. Officially, this is my understanding.

**Comment by bwoodsend** [comment]
@Xiaozhi-Python Could you stop pinging random old issues? It's not particularly constructive (even when you don't misinterpret the original issue and post something unrelated).

**Comment by XiaoZhi-Python-China** [comment]
> @小智-Python你能不能不要再pinging随机老问题了？也不是特别有建设性(即使你没有曲解原问题，发些无关的东西)。

No, it's none of your business if I don't ask questions. You think it's awesome. If you encounter literature and art that you can't, you have to ask. Learning is not a quarrel. The affordable teacher who went to school didn't teach you. Don't be shameless. Don't say I bully you when I make the pyinstaller auxiliary tool.
